chia/full_node/sync_store.py

```diff
@@ -93,38 +88,34 @@ def get_peers_that_have_peak(self, header_hashes: List[bytes32]) -> Set[bytes32]
                     node_ids.add(node_id)
         return node_ids
 
-    def get_peak_of_each_peer(self) -> Dict[bytes32, Tuple[bytes32, uint32, uint128]]:
+    def get_peak_of_each_peer(self) -> Dict[bytes32, Peak]:
         """
         Returns: dictionary of peer id to peak information.
         """
 
         ret = {}
-        for peer_id, v in self.peer_to_peak.items():
-            if v[0] not in self.peak_to_peer:
+        for peer_id, peak in self.peer_to_peak.items():
+            if peak.header_hash not in self.peak_to_peer:
```

Comment:
i know this is not related to the pr but this discrepancy between the two Dictionaries is wrong
i guess we ignore ones not in peak_to_peer because there is no way of finding them in peer_to_peak
but maybe we should add a warning ?
i dont think this should happen unless something went wrong

Reply:
Yeah thats unrelated to here, i will put it on my list, might be worth investigating later.